examples/pimlico/src/account.js

```diff
@@ -21,7 +21,6 @@ export const getAccount = async (type) => {
             const simpleAccount = await toSimpleSmartAccount({
                 owner,
                 client,
-                factoryAddress: "0x9406Cc6185a346906296840746125a0E44976454",
```

Comment:
factoryAddress only created v06 compatible wallets. Removing it defaults it to creating a new v07 wallet.